### DISCUSSION TITLE
ImPlotFlags_AntiAliased

### DISCUSSION THREAD

**Discussion by tmsampson** [epezent/implot#491] [Q&A] [answered]
I'm currently upgrading an old project to use the latest version of ImPlot, which is currently failing due to apparent removal of **_ImPlotFlags_AntiAliased_** flag from **_ImPlotFlags__** enum. 

However, the README still mentions using **_ImPlotFlags_AntiAliased_** (or setting **_ImPlot::GetStyle().AntiAliasedLines_**), both of which now seem to be gone. So, a couple of questions...

1) Is the README out of date? 
2) If this functionality has been removed, is there an alternative method for achieving AA rendering of the plots? I'm not that familiar with the code I'm upgrading and just looking to achieve parity with the old version where possible....

TIA - Maybe I'm missing something obvious here...

**Comment by brenocq** [comment, marked as answer]
The `README` was updated about this in commit 35cf7092dba2bb91373a6bbab8c8efabe287cb55.

I'm closing this discussion because it is no longer relevant in the latest version (the `README` explains how to enable AA)
